cli/cmd/alert_profiles.go

```diff
@@ -150,6 +151,25 @@ Then go to Settings > Alert Profiles.
 			return nil
 		},
 	}
+
+	// update command is used to update an existing lacework alert profile
+	alertProfilesUpdateCommand = &cobra.Command{
+		Use:   "update",
+		Short: "Update an existing alert profile",
+		Args:  cobra.MaximumNArgs(1),
+		RunE: func(_ *cobra.Command, args []string) error {
+			if !cli.InteractiveMode() {
+				return errors.New("interactive mode is disabled")
+			}
+			response, err := promptUpdateAlertProfile(args)
+			if err != nil {
+				return errors.Wrapf(err, "unable to update alert profile %s", response.Data.Guid)
```

Comment:
no, let's not access `response.Data.Guid` if there is an error, it could be `nil`

Reply:
```suggestion
				return err
```